bc/settings/django.py

```diff
@@ -45,8 +45,6 @@
 
 TAILWIND_APP_NAME = "bc.web"
 
-INTERNAL_IPS = ("127.0.0.1",)
```

Comment:
Is this no longer set when `DEVELOPMENT is False`? It looks like it's still set when `DEVELOPMENT is True`, but not when it's `False`?